scripts/index.js

```diff
@@ -1 +1,367 @@
+
+const arrayOfNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+// CREATING ARRAY FOR FUNCTION^
+
+
+function isPositive(number) {
+    return number >= 0
+};
+// CREATING HELPER FUNCTION(Determines if number isPositive)^
+
+
+const arrayOfPositives = arrayOfNumbers.filter(isPositive);
+// CALLING FILTER METHOD^
+
+
+console.log(`#1`)
+console.log(arrayOfPositives);
+
+
+
+
+
+// 2. Even Numbers
+// -----------------
+    // Write a function which takes an array of numbers as input and returns a new array containing only the even numbers in the given array.
+//________________________________________________________________________________________________________________________________
+
+// const arrayOfNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]; // (created in # 1 ^)
+
+
+function isEven(number) {
+    return number % 2 === 0 
+}
+// CREATING HELPER FUNCTION(Determines if number isEven)^
+
+
+const arrayOfEvens = arrayOfNumbers.filter(isEven);
+// CALLING FILTER METHOD^
+
+
+console.log(`#2`)
+console.log(arrayOfEvens);
+
+
+
+
+
+// 3. Square the Numbers
+// ---------------------
+    // Write a function which takes an array of numbers as its argument and returns a new array containing result of squaring each of the numbers in the given array by two. Example: squareTheNumbers([1, 2, 3]) should give [1, 4, 9].
+//________________________________________________________________________________________________________________________________
+
+// const arrayOfNumbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]; // (created in # 1 ^)
+
+
+function squareTheNumbers(number) {
+    return number * number;
+};
+// CALLING MAP & FILTER METHOD^
+
+const arrayOfSquares = arrayOfNumbers.map(squareTheNumbers);
+// CALLING MAP METHOD:^
+
+
+console.log(`#3`)
+console.log(arrayOfSquares);
+
+
+
+
+
+// 4. Cities 1
+// --------------
+
+    // Write a function which takes an array of city objects like such:
+
+        // const cities = [
+        //   { name: 'Los Angeles', temperature: 60.0},
+        //   { name: 'Atlanta', temperature: 52.0 },
+        //   { name: 'Detroit', temperature: 48.0 },
+        //   { name: 'New York', temperature: 80.0 }
+        // ];
+    
+    // as input and returns a new array containing the cities whose temperature is cooler than 70 degrees.
+//________________________________________________________________________________________________________________________________
+
+const cities = [
+          { name: 'Los Angeles', temperature: 60.0},
+          { name: 'Atlanta', temperature: 52.0 },
+          { name: 'Detroit', temperature: 48.0 },
+          { name: 'New York', temperature: 80.0 }
+        ];
+// CREATING ARRAY FOR FUNCTION^
+
+function removeCitiesHotterThan70(item) {
+    return item.temperature <= 70
+};
+// CREATING HELPER FUNCTION(Filters list of only objects with temps<70)^
+
+
+function getCitiesCoolerThan70(item) {
+    if (item.temperature <= 70) {
```

Comment:
Interesting! It seems that you're checking twice if the city's temperature is `<=` 70.
If the `.filter` is already handling that, maybe you don't need do that in this helper function?

(That is, if you `.filter` out the hot cities first, the `.map` will only be dealing with cities that are cooler than 70.)